 <launch>
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="base_side" args="0 -1.375 0 0 0 0 base_link_side base_link"/> -->
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="base_side" args="0.57 0 0 0 0 0 base_link_gps base_link"/> -->

    <node pkg="keyboard" type="keyboard" name="keyboard" output="screen"/> 
    <!-- <include file="$(find teleop_twist_joy)/launch/teleop.launch"/> -->

    <node pkg="gps" type="gps_base" name="gps_base">
        <param name="sim_flag" value="false"/>
        <param name="setMapZeroFlag" value="false"/>
        <param name="utmx_zero" value="440575.08370527084"/>
        <param name="utmy_zero" value="4423940.761659693"/>
        <!-- car 000 -->
        <!-- <param name="angle_bias" value="0.0"/>
        <param name="x_bias" value="0.89"/>
        <param name="y_bias" value="0.00"/> -->
        <!-- car 001 -->
        <!-- <param name="angle_bias" value="-0.2"/>
        <param name="x_bias" value="0.87"/>
        <param name="y_bias" value="-0.01"/> -->
        <!--串口和以太网只能选择一个(串口)-->
        <!-- <param name="mode" value="SER"/>
        <param name="devname" value="/dev/ttyS3"/>
        <param name="baud" value="115200"/> -->

        <!--串口和以太网只能选择一个(以太网)1234-->
        <param name="mode" value="UDP"/>
        <param name="IP" value="192.168.2.2"/>
        <param name="port" value="1234"/> 
    </node>
 </launch>
